internal/pkg/jwt/jwt.go

```diff
@@ -0,0 +1,114 @@
+package jwt
+
+import (
+	"fmt"
+	"github.com/TheFlies/ofriends/internal/pkg/config/env"
+	"github.com/dgrijalva/jwt-go"
+	"log"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type (
+	Customeclaims struct {
+		jwt.StandardClaims
+		Useid       string
+		Usefullname string
+		Userole     string
+	}
+	JWTConf struct {
+		PrivateKey string `envconfig:"PRIVATE_KEY"`
+		TimeOut    string `envconfig:"TIME_OUT"`
+	}
+)
+
+func New(standard jwt.StandardClaims, uuseid string, usefullname string, useroole string) *Customeclaims {
```

Comment:
let take a look at Go name convention and correct these.  Should be userID, userFullName, userRole